src/server/accounting.c

```diff
@@ -1190,6 +1197,7 @@ account_jobend(job *pjob, char *used, int type)
 		if ((used = strdup(tmp_resc_used)) == NULL)
 			goto writeit;
 		pjob->ji_acctrec = used;
+		free_attrlist(&temp_head);
```

Comment:
What is the purpose of temp_head?  You set initialize, set, and free it, but never use it.  Is it just there to make the encode function happy?